.github/ISSUE_TEMPLATE/variable-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Variable request
+about: Suggest a variable for cchsflow
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Hmm. I think we have different levels that we should consider. 
People may just want to make a request and not do too much work. An intro something like:
"We accept requests and PR for new variables. Providing information about your request helps discussion about whether and how to include the variable."